src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -402,8 +402,6 @@ The .NET Foundation licenses this file to you under the MIT license.
       <CompileWasmArgs Condition="'$(IlcLlvmExceptionHandlingModel)' == 'cpp'">$(CompileWasmArgs) -s DISABLE_EXCEPTION_CATCHING=0</CompileWasmArgs>
       <CompileWasmArgs Condition="'$(IlcLlvmExceptionHandlingModel)' == 'wasm'">$(CompileWasmArgs) -fwasm-exceptions</CompileWasmArgs>
       <CompileWasmArgs Condition="'$(NativeDebugSymbols)' == 'true'">$(CompileWasmArgs) -g3</CompileWasmArgs>
-      <!-- TODO-LLVM: remove -s USE_SDL=0 when we update to Emscripten with https://github.com/emscripten-core/emscripten/pull/18443 -->
-      <CompileWasmArgs>$(CompileWasmArgs) -s USE_SDL=0 $(WasmOptimizationSetting)</CompileWasmArgs>
```

Comment:
This also deletes `$(WasmOptimizationSetting)`. Why would that make the tests fail in such a way, though...

Reply:
THanks, I'll put that back.  Put looking at why it crashes, this is the wat
```wat
  (func $S_P_CoreLib_System_Environment___cctor (;839;) (param $var0 i32)
    (local $var1 i32)
    (local $var2 i32)
    (local $var3 i32)
    (local $var4 i32)
    i32.const -4
    local.set $var1
    local.get $var1
    i32.load offset=909728
```

I can't find in the spec where it says the dynamic arg `-4` is not valid, but it appears so playing with (module

```wat
  (memory $memory 1)
  (export "memory" (memory $memory))

  (func (export "load_first_item_in_mem") (param $num i32) (result i32)
    i32.const -4

    ;; load first item in memory and return the result
    i32.load offset=4
  )

)
```
in https://developer.mozilla.org/en-US/docs/WebAssembly/Reference/Memory/Load.  An emscripten bug somewhere (binaryen/clang maybe)?  Strangley, locally the debug tests are ok, only the release fail.

Reply:
It sure does look like some strange LLVM bug, but bad (or just slightly unorthodox) codegen on our part is also not out of the question. Even more strange, in `browser-wasm windows Debug` a libraries test fails, even as for `-c Debug` this change shouldn't matter.

> I can't find in the spec where it says the dynamic arg -4 is not valid, 

I think load/stores are meant to take the argument as unsigned, so `-4` is basically trying to access the last few bytes of all possible linear memory (even not taking into account that with the large offset, the final address computation would overflow).

I think we can keep this PR scoped to `USE_SDL` and open a follow-up issue if restoring the optimization setting restores the CI.